php/php.editor/src/org/netbeans/modules/php/editor/indent/FormatVisitor.java

```diff
@@ -2175,7 +2193,12 @@ public void visit(SingleFieldDeclaration node) {
                 assert (parent instanceof FieldsDeclaration);
                 FieldsDeclaration fieldsDeclaration = (FieldsDeclaration) path.get(1);
                 if (ts.token().id() == PHPTokenId.PHP_OPERATOR && TokenUtilities.textEquals("=", ts.token().text())) { //NOI18N
-                    int realNodeLength = fieldsDeclaration.getModifierString().length() + " ".length() + name.getEndOffset() - name.getStartOffset(); //NOI18N
+                    // GH-7190
+                    int fieldTypeLength = getTypeLength(fieldType);
+                    if (fieldTypeLength != 0) {
+                        fieldTypeLength += " ".length();
```

Comment:
As above, perhaps using a constant named e.g. `SPACE_LENGTH` could be a good idea here.

Reply:
I'll do that. Thank you, Tomas!

Reply:
It is just a nitpick, feel free to ignore it. I think that JVM optimizes it :)

Reply:
Fixed: https://github.com/apache/netbeans/compare/cce07cb62abab658d6447f7a25af1aebbfe5cf79..f586da1b03743c847cdc63fb37307acbc912ec2f